src/EchoToImageLUT.cpp

```diff
@@ -129,4 +129,17 @@ void EchoToImageLUT::drawImageFromEchoes(std::vector<uint8_t> const& world_echoe
     for (long i = 0; i < static_cast<long>(world_echoes.size()); i++) {
         updateImage(radar_frame, i / m_sweep_size, i % m_sweep_size, world_echoes.at(i));
     }
-}
\ No newline at end of file
+}
+
+std::vector<cv::Point> EchoToImageLUT::getPixelsFor(int angle_unit,
```

Comment:
1. keep names uniform

```suggestion
std::vector<cv::Point2i> EchoToImageLUT::getPixelsFor(int angle_idx,
```

2. Another idea would be to return a pair of iterators for the points range something like

```cpp
...
return std::make_pair(m_data.cbegin() + point_idx, m_data.cbegin() + net_point_idx);
```
and then use:
```cpp
for(auto [pixel, last_pixel] = getPixelsFor(theta_i, sweep_j); pixel < last_pixel; pixel++) {
  // do some stuff with *pixel
}
```

Reply:
good call